### ISSUE TITLE
Create standard "people in" table

### ISSUE THREAD

**Issue by msackton** [tsackton/taelgar#15]
This, unlike the events and leadersOf tables, is meant to be used for session prep and in-Obsidian data exploration and consistency only.

The idea is a dv.view that can be embedded on a page to show all of the people who are either based in that place, or currently in that place, or last known to be in that place.

This is especially useful when running a game with 20+ NPCs who have been made up in a town, but where it is easy to forget exactly who has left or not left the town. By being good about updating whereabouts, it becomes straightforward to see who, at a given date, is actuallly in the place.

This is not expected to be useful for export, as you wouldn't usually want to see a list of all the people who live in a town in the export.

**Comment by msackton** [comment]
A future idea could be to have the idea of "notable" people, i.e. some tag (person/notable? person/notable/chardon?) which would cause a person to be considered "notable" in a particular location.

This would allow automatic generation of a "Notable People" section for places, which might be nice. 

This idea doesn't scale super well to multiple dates though because it seems quite complex to define a "notableAt date" and it isn't clear that it would be useful to export a table of noteable people that contains 2 year olds or 87 year olds, neither of whom are necessarily noteable at those ages.

It might be worth considering at some future point though, if for example both DuFr and Clee spend significant time in Tollen so it is worth thinking about how to manage the differences in 1720s and 1740s Tollen

**Comment by tsackton** [comment]
I actually think this is potentially more useful for export than the events page. My sense is when you are trying to remember NPCs, it is often "that X in place Y", and having a quick summary of who you met in Tollen potentially the fastest way to quickly find what you need. 

I'm not sure there is a simple way to do this, without backfilling lots of campaign info information into NPCs and PCs, or perhaps adding location tags to sections of session notes (or having a "NPCs" section of session notes that could be parsed or something). 

If I were starting fresh, I'd change my session note prep to be less about writing a narrative and more about structured tagging of what happened.

**Comment by msackton** [comment]
Right, I think that's a feature that seems conceptually easily to build than notable, because it can be inside a %%Campaign:%% block so you don't have to worry about differences between time periods. You are not outputting a notable people list, you are outputting a list of people in a place that the party knows of.

But actually I think that might be better as just a generated page -- index of NPCs or something.

I've been manually maintaining one (Cleenseau Campaign - Index of NPCs) and my players do reference that at times to remind themselves of people.

You could imagine just auto-generating that page. From the perspective of a player I think I'd rather just have an index of people we know with the last known location, rather than the information embedded on a page.

If we ended up in Tokra again, say, what I'd want to know is:

* who are the important people we've heard of (see noteable people above) and
* who are the people we know who might be in Tokra
